draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -619,44 +630,47 @@ TBD
 ## JSON Web Token Claims Registration
 
 - Claim Name: "vct"
-  - Claim Description: Verifiable credential type identifier
-  - Change Controller: IETF
-  - Specification Document(s): (#type-claim) of this document
+- Claim Description: Verifiable credential type identifier
+- Change Controller: IETF
+- Specification Document(s): (#type-claim) of this document
 
 ## Media Types Registry
 
 ### application/vc+sd-jwt {#application-vc-sd-jwt}
 
 The Internet media type for a SD-JWT VC is `application/vc+sd-jwt`.
 
-Type name: : `application`
-
-Subtype name: : `vc+sd-jwt`
-
-Required parameters: : n/a
-
-Optional parameters: : n/a
+* Type name: `application`
+* Subtype name: `vc+sd-jwt`
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: 8-bit code points; SD-JWT VC values are encoded as a series of base64url-encoded values (some of which may be the empty string) separated by period ('.') and tilde ('~') characters.
+* Security considerations: See Security Considerations in (#security-considerations).
+* Interoperability considerations: n/a
+* Published specification: [[ this specification ]]
+* Applications that use this media type: Applications that issue, present, and verify SD-JWT-based verifiable credentials.
+* Additional information:
+  - Magic number(s): n/a
+  - File extension(s): n/a
+  - Macintosh file type code(s): n/a
+* Person & email address to contact for further information: Oliver Terbu <oliver.terbu@mattr.global>
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: Oliver Terbu <oliver.terbu@mattr.global>
+* Change controller: IETF
 
-Encoding considerations: : 8-bit code points; SD-JWT VC values are encoded as a
-series of base64url-encoded values (some of which may be the empty string)
-separated by period ('.') and tilde ('~') characters.
+## Well-Known URI Registry
 
-Security considerations: : See Security Considerations in (#security-considerations).
+This specification requests the well-known URI defined in (#jwt-vc-issuer-metadata)
+in the IANA "Well-Known URIs" registry [@IANA.well-known] established
+by [@!RFC5785].
 
-Interoperability considerations: : n/a
+### Registry Contents
 
-- Published specification: : RFC TODO
-- Applications that use this media type: : Applications that issue, present,
-  verify verifiable credentials and presentations.
-- Additional information:
-  - Magic number(s): n/a
-  - File extension(s): n/a
-  - Macintosh file type code(s): n/a
-  - Person & email address to contact for further information: TBD
-  - Intended usage: COMMON
-  - Restrictions on usage: none
-  - Author: Oliver Terbu <oliver.terbu@mattr.global>
-  - Change controller: IETF
+* URI suffix: jwt-vc-issuer
+* Change controller: IETF
+* Specification document: (#jwt-vc-issuer-metadata) of [[ this of this document ]]
```

Comment:
in SD-JWT the section is also included like this, not sure if there was a really good reason for that. In case it was an oversight, we would need to do the following:

```suggestion
* Specification document: [[ (#jwt-vc-issuer-metadata) of  this of this document ]]
```